### PR TITLE
fix: [FFM-5360]: Updating operator causing Env not to be created

### PR DESCRIPTION
**Pull request by laura-mannus**
### Summary

A double operator was causing the value to be converted to a boolean (false) and then to (true), so causing the create env dialogue to not appear.

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by crystal-fong — DISMISSED** [review]

File: src/modules/75-cf/pages/onboarding/views/SelectEnvironmentView.tsx [hunk 1]
```diff
@@ -95,7 +95,7 @@ export const SelectEnvironmentView: React.FC<SelectEnvironmentViewProps> = ({
               <Container width={250}>
                 <EnvironmentSelect />
               </Container>
-              {!!environments?.length && createEnvName && dialogOpen && (
+              {createEnvName && dialogOpen && (
```
Comment (crystal-fong) [inline src/modules/75-cf/pages/onboarding/views/SelectEnvironmentView.tsx:98]: will it be grand if `environments` check is gone?

**Review by knagurski — DISMISSED** [review]
Looks good 👍🏻

**Review by crystal-fong — APPROVED** [review]

File: src/modules/75-cf/pages/onboarding/views/SelectEnvironmentView.tsx (hunk 1 above)
Comment (laura-mannus) [reply src/modules/75-cf/pages/onboarding/views/SelectEnvironmentView.tsx:98]: yeah been speaking to @knagurski we think this is the best way :)

Comment (laura-mannus) [reply src/modules/75-cf/pages/onboarding/views/SelectEnvironmentView.tsx:98]: my fix previously didn't fix it for all scenarios

**Review by crystal-fong — APPROVED** [review]

**Pull request merged**